### PR TITLE
Add a FAQ help page for MySQL issue about PERMISSIVE mode.

### PR DESCRIPTION
**Pull request by incfly**
Doc part for istio/istio#10062

### CODE REVIEW

File: content/help/faq/security/mysql-with-mtls.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: MySQL Connectivity Troubleshooting
+description: Integrate VMs and bare metal hosts into an Istio mesh deployed on Kubernetes.
+weight: 95
+keywords: [mysql,mtls]
+---
+
+# MySQL Connectivity Troubleshooting
+
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: No need for this header, the infrastructure supplies the title based on the content of title: above

File: content/help/faq/security/mysql-with-mtls.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+kind: "Policy"
+metadata:
+  name: mysql-nomtls-authn
+spec:
+  targets:
+  - name: mysql-service
+EOF
+```
+
+## Enable Mutual TLS by STRICT mode
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: by -> in

File: content/help/faq/security/mysql-with-mtls.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+default option `istio-demo.yaml` install, does not work with MySQL.
+
+We have two options to solve the problem.
+
+## Disable Mutual TLS
+
+Choose this option if you don't want Istio mutual TLS. You achieve this by disable mutual TLS on MySQL
+service explicitly.
+
+```bash
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: Please use a {{< text >}} block here. See https://istio.io/create for details.

File: content/help/faq/security/mysql-with-mtls.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  targets:
+  - name: mysql-service
+EOF
+```
+
+## Enable Mutual TLS by STRICT mode
+
+If you want mutual TLS protection for MySQL, enable mutual TLS via `DestinationRule` and `AuthenticationPolicy`.
+
+```bash
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: {{< text >}} block here too.

File: content/help/faq/security/mysql-with-mtls.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  name: mysql-nomtls-authn
+spec:
+  targets:
+  - name: mysql-service
+EOF
+```
+
+## Enable Mutual TLS by STRICT mode
+
+If you want mutual TLS protection for MySQL, enable mutual TLS via `DestinationRule` and `AuthenticationPolicy`.
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: ... mutual TLS using a destination rule and an authentication policy.

File: content/help/faq/security/mysql-with-mtls.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+# MySQL Connectivity Troubleshooting
+
+You may find MySQL can't be connected after installing Istio. This is because that `PERMISSIVE` mode,
+default option `istio-demo.yaml` install, does not work with MySQL.
+
+We have two options to solve the problem.
+
+## Disable Mutual TLS
+
+Choose this option if you don't want Istio mutual TLS. You achieve this by disable mutual TLS on MySQL
```
Comment (geeknoid) [inline content/help/faq/security/mysql-with-mtls.md]: by disabling

Comment (geeknoid) [reply content/help/faq/security/mysql-with-mtls.md]: on MySQL -> on the MySQL

File: content/help/faq/security/mysql-with-mtls.md (hunk 1 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Removed.

File: content/help/faq/security/mysql-with-mtls.md (hunk 2 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Done.

File: content/help/faq/security/mysql-with-mtls.md (hunk 3 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Done. Just curious what's the benefits of using {{<text>}}? Is that a syntax we create ourselves?

File: content/help/faq/security/mysql-with-mtls.md (hunk 4 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Done.

File: content/help/faq/security/mysql-with-mtls.md (hunk 5 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Done.

File: content/help/faq/security/mysql-with-mtls.md (hunk 6 above)
Comment (incfly) [reply content/help/faq/security/mysql-with-mtls.md]: Done.

**Comment by incfly** [comment]
All comments resolved, PTAL, thanks!

**Comment by geeknoid** [comment]
@rcaballeromx ?

**Comment by geeknoid** [comment]
Note the CI failures:

./content/help/faq/security/mysql-with-mtls.md:34: MD012 Multiple consecutive blank lines
./content/help/faq/security/mysql-with-mtls.md:13: MD036 Emphasis used instead of a header
./content/help/faq/security/mysql-with-mtls.md:30: MD036 Emphasis used instead of a header

**Comment by incfly** [comment]
Lint fixed. Using headlines will result user unfriendly format. Changed to orderded list instead.
@geeknoid @rcaballeromx

**Comment by incfly** [comment]
ping @geeknoid and @rcaballeromx

File: content/help/faq/security/mysql-with-mtls.md (hunk 3 above)
Comment (geeknoid) [reply content/help/faq/security/mysql-with-mtls.md]: {{< text >}} is custom syntax we use. If does error checking for a few cases, plus supports options like downloadas outputis, snippet, and expandlinks

**Pull request merged**